central/graphql/resolvers/plotted_node_vulnerabilities.go

```diff
@@ -49,7 +75,41 @@ func (pvr *PlottedNodeVulnerabilitiesResolver) BasicNodeVulnerabilityCounter(_ c
 
 // NodeVulnerabilities returns the node vulnerabilities for top risky nodes scatter-plot
 func (pvr *PlottedNodeVulnerabilitiesResolver) NodeVulnerabilities(ctx context.Context, args PaginatedQuery) ([]NodeVulnerabilityResolver, error) {
```

Comment:
The graphQL field is `"nodeVulnerabilities(pagination: Pagination): [NodeVulnerability]!",` but the resolver is accepting full query—query string and pagination. Update this resolver sig to only accept the pagination.

Reply:
go for it